muikku-core-plugins/src/main/resources/META-INF/resources/dust/guider/guider_view_profile.dust

```diff
@@ -40,6 +40,7 @@
 <!--       {/senderHasPicture} -->
 <!--     </div> -->
     <div class="gt-user-details-items lg-flex-cell-full md-flex-cell-full sm-flex-cell-full ">
+    <div class="gt-tool-hide-profile mf-item-tool-btn"><span>Back</span></div>
```

Comment:
This needs to be localized as the content of the span should function as tooltip.